courses/web/03-Web-In-Practice-1-Client.md

```diff
@@ -36,3 +36,4 @@
 10. Find at least 5 other online resources about client side JS & Tooling, and explain why you chose them. 
 11. Exactly **7 days** before your deadline, ask your mentor to arrange a twitch time for you on your deadline day.
 12. You should give a 20-min presentation about the content of this step, on Twitch platform.
+13. You are supposed to become mentor and start to help another interns to study courses and improve their soft skills.
```

Comment:
13. It's time to be a mentor. Ask your mentor to start your onboarding process for mentorship.
@nfsdns